### DISCUSSION TITLE
Provide update instructions

### DISCUSSION THREAD

**Discussion by s0enke** [superwerker/superwerker#203] [Ideas] [answered]
Currently, superwerker does not provide docs on how to upgrade. 

As an example, M1.1 release instructions on the mailing list: https://groups.google.com/g/superwerker/c/2XiwygOcvH0

## TODO

- Extend README.md
  1. Get CloudFormation template URL for new version 
  2. Access CloudFormation console
  3. Update CloudFOrmation stack
- Extend GitHub Version description with link to update quide in README.md

## Important

The linked guide is missing an important step:

* Change the parameter QSS3KeyPrefix to the current version.

When, for example, there is a superwerker installation from `main` or `0.0.1` (or any other version) and you update the cloudformation stack with a new template file ( `https://superwerker-deployment.s3.amazonaws.com/0.12.2/templates/superwerker.template.yaml` for example ), the `QSS3KeyPrefix` has still the previous value. So an update following the mentioned steps will only update the `superwerker.template.yml` file but not the nested stacks.

**Comment by sbstjn** [comment]
@Reb-B updated the `README.md` with the steps needed to update superwerker.

**Comment by s0enke** [comment, marked as answer]
Link to update instructions in release notes provided here: https://github.com/superwerker/superwerker/commit/29aa10deebaa0702f42831baa6b17e3b015c88b9
